modules/serverless-admin-monitoring-eventing-broker-trigger.adoc

```diff
@@ -19,16 +19,17 @@ You can use the {product-title} monitoring dashboards to view metrics for broker
 . Select the *Knative Eventing - Broker/Trigger* dashboard in the *Dashboard* drop-down list.
 . You can now view the following metrics:
 .. For brokers:
-*** Event Count (rate per minute)
-*** Success Rate (2xx Event, rate per minute)
-*** Event Count by Event Type (rate per minute)
-*** Failure Rate (non-2xx Event, fraction of rate per minute)
-*** Event Count by Response Code Class (rate per minute)
+*** Event Count (avg/sec, over 1m window)
+*** Success Rate  (2xx Event, fraction rate, over 1m window)
```

Comment:
do we use two spaces by intention ? or by accident?

Reply:
@matzew Good catch. I copied this from https://github.com/openshift-knative/serverless-operator/pull/1044/files , I left a comment for @skonto regarding it.